src/actions/index.js

```diff
@@ -1,9 +1,9 @@
 export { createGithubRelease } from './createGithubRelease'
 export { createReleaseNotes } from './createReleaseNotes'
-export { evaluateNewVersionByPackage } from './evaluateNewVersionByPackage'
+export { evaluateNewVersionsByPackage } from './evaluateNewVersionsByPackage'
```

Comment:
Sorry if I introduced the idea of `versions` by package. But there is only one new version for each package so it should be `evaluateNewVersionByPackage`.

And the other one is `getCurrentVersions` (without by package).